generators/app/templates/common/README.md

```diff
@@ -87,6 +87,17 @@ make install
 ```
 <% if (includeApi && apiInfrastructure === "aws") { -%>
 
+### Setup environment variables
+
+Duplicate the `.env.example` file and rename it to `.env`. Fill in the environment variables with the right values.
+
+Make sure to install [direnv](https://github.com/direnv/direnv/tree/master) to load the environment variables automatically whenever you enter the project directory.
+On MacOS, you can install it with Homebrew:
+```bash
+brew install direnv
+```
+
+
```

Comment:
Does it respond to all use cases for loading var env ? 
-> Will streamlit / poetry benefit from it ? 
-> Does your IDE need additionnal config ? 

My understanding, is that is only apply for local terminal -> Any production setup like a docker image needs to handle it itself.

Reply:
- The variables are loaded in the environment, so yes it benefits poetry and streamlit when they are run from the terminal
- No, actually the well-configured IDE already add the current workdir to the PYTHONPATH, but direnv makes sure that even the terminals outside an IDE have the correct PYTHONPATH

And yes it only applies for local terminal